index.js

```diff
@@ -204,6 +205,16 @@ export default function normalizeUrl(urlString, options) {
 		urlObject.search = '';
 	}
```

Comment:
line 195 needs a check for `options.keepQueryParameters`, to avoid useless work.

Reply:
Added length check for `options.keepQueryParameters` (f8a53032f6c6c91de1804e6d5328457009622f5b)